src/PlaywrightSharp.Tests/BrowserContextAddCookiesTests.cs

```diff
@@ -44,7 +44,17 @@ public async Task ShouldRoundtripCookie()
             var cookies = await Context.GetCookiesAsync();
             await Context.ClearCookiesAsync();
             Assert.Empty(await Context.GetCookiesAsync());
-            await Context.AddCookiesAsync(cookies.Select(c => (SetNetworkCookieParam)c));
+            await Context.AddCookiesAsync(cookies.Select(c => new Cookie()
+            {
+                Domain = c.Domain,
+                Expires = c.Expires,
+                HttpOnly = c.HttpOnly,
+                Name = c.Name,
+                Path = c.Path,
+                SameSite = c.SameSite,
+                Secure = c.Secure,
+                Value = c.Value
+            }));
```

Comment:
Can't we make an implicit conversion?

Reply:
I've removed `SetNetworkCookieParam` class, and would rather keep it explicit for the user to **know** there is a change. It's not _that_ painful; should just be a rename in the code.